fixtures/artifact_bundle_debug_ids/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "org": "__org__",
+  "release": "__release__",
+  "files": {
+    "files/_/_/index.js.map": {
+      "url": "~/index.js.map",
+      "type": "source_map",
+      "headers": {
+        "debug-id": "Eb6e60f1-65ff-4f6f-Adff-f1bbeDEd627b"
+      }
+    },
+    "files/_/_/index.js": {
+      "url": "~/index.js",
+      "type": "source"
+    },
+    "files/_/_/index.min.js": {
+      "url": "~/index.min.js",
+      "type": "minified_source",
+      "headers": {
+        "Debug-id": "Eb6e60f1-65ff-4f6f-Adff-f1bbeDEd627b",
```

Comment:
I guess its intentional that you have different casing here?
(I believe we should have an explicit test to make sure we can correctly consume these no matter the casing)

Reply:
We agreed with @mitsuhiko to keep it for testing multiple cases but I agree, I would like to test all the new code with ad-hoc edge cases properly.

I don't expect to merge this PR without having first clarified source maps lookups. I aim to have a prototype of everything working and then refine everything as needed.